src/gam.py

```diff
@@ -5346,8 +5346,12 @@ def transferDriveFiles(users):
 
 def sendOrDropEmail(users, method='send'):
   body = subject = ''
-  recipient = labels = sender = None
+  recipient = sender = None
+  labels = []
```

Comment:
We should keep this as None so that if labels are not set we don't set the labels argument on the API call at all.